test/invariants/fuzz/BreadfundFuzz_DepositWithdraw.t.sol

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+/**
+ * ────────────────────────────────────────────────────────────────────────────────
+ * BreadfundFuzz_DepositWithdraw.t.sol
+ *
+ * Purpose: Fuzz the core deposit/withdraw lifecycle. Mixes many deposits,
+ * small withdrawals, large withdrawals, request execution, and epoch progress
+ * to stress accounting invariants.
+ *
+ * Properties checked:
+ *  - Deposits only once per epoch per member (`AlreadyDeposited`).
+ *  - Withdrawals respect daily contribution, thresholds, and per-epoch limits.
+ *  - Large withdrawals generate requests (and execute only after contest window).
+ *  - Soft conservation invariant (`contract balance + executed ≥ withdrawables`)
+ *    holds at each step.
+ *  - Epoch warp advances counters/reset correctly.
+ *
+ * Notes for reviewers:
+ *  - Some checks are unit-like but are *critical safety guards* (e.g.,
+ *    “already deposited this epoch”).
+ *  - After fixing BUG #2 (large exec not decrementing withdrawables),
```

Comment:
need to make note of this somewhere besides comments here. include note in issue thats created for this bug